packages/activation-notice/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@web-stories-wp/activation-notice",
+  "private": true,
+  "engines": {
+    "node": ">= 15",
+    "npm": "~7"
+  },
+  "exports": {
+    "default": "./src/index.js"
+  },
+  "main": "./src/index.js",
+  "type": "module",
+  "dependencies": {
+    "@wordpress/i18n": "^3.17.0",
+    "@web-stories-wp/i18n": "*",
+    "@web-stories-wp/tracking": "*",
+    "flagged": "^2.0.1",
+    "polished": "^4.0.5",
+    "prop-types": "^15.7.2",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
+    "styled-components": "^5.2.1"
+  },
+  "devDependencies": {
+    "stylis-plugin-rtl": "^1.0.0",
```

Comment:
That's not a dev dependency. It belongs to `dependencies` above.

Reply:
Good spot. https://github.com/google/web-stories-wp/pull/6278/commits/10df4ff3a2e084edf4ac0fcdc6ba580552ea78c3